### ISSUE TITLE
[Proposal] Add 'ReturnDatesAsStrings' option to Sql Server connection

### ISSUE THREAD

**Issue by codedge** [laravel/ideas#649]
As the new PHP for SQL Driver [support returning dates as string](https://docs.microsoft.com/en-us/sql/connect/php/how-to-retrieve-date-and-time-type-as-strings-using-the-sqlsrv-driver) it would be nice to pass this option via config to the [`getSqlSrvDsn()` method](https://github.com/laravel/framework/blob/43bda8441f23349086aaed239bdf1af58f2eb860/src/Illuminate/Database/Connectors/SqlServerConnector.php#L99).

Basically in your `config/database.php` you would specify sth. like

```php
 'connections' => [

        'mssql' => [
            'driver' => 'sqlsrv',
            //...
            'return_dates_as_string' => true
        ],
        //...
]
```

**Comment by orditeck** [comment]
Why was this closed? Is this added @taylorotwell?

**Comment by RipleWare** [comment]
Is there any update to this?  I have come across the need for this option to.
